src/handlers/zapierTriggers.ts

```diff
@@ -0,0 +1,20 @@
+import { fetchStatuses } from './validators';
+import ky from "ky";
```

Comment:
This currently throws an error because `ky` is not added as a dependency, but also see below.